src/writer.jl

```diff
@@ -487,13 +487,15 @@ function write_parquet(io::IO, x; compression_codec = "SNAPPY")
 
     # figure out the right number of chunks
     # TODO test that it works for all supported table
-    table_size_bytes = Base.summarysize(tbl)
+    nrows = length(Tables.rows(tbl))
```

Comment:
```suggestion
    nrows = Tables.rowcount(tbl)
```
This is a mostly public Tables.jl function that only works on the result of `Tables.columns`, which `tbl` is here as we've called for above.